interfaces/cython/cantera/reactor.pyx

```diff
@@ -388,6 +388,176 @@ cdef class FlowReactor(Reactor):
+        total number of surface species in all surfaces.
+
+    ``get_surface_initial_condition(self, y : double[:]) -> None``
+        Responsible for populating the state vector ``y`` with the initial
+        state of each surface phase in this reactor. The length of ``y`` is the
+        total number of surface species in all surfaces.
+
+    ``update_connected(self, update_pressure : bool) -> None``
+        Responsible for storing properties which may be accessed by connected
+        reactors, and for updating the mass flow rates of connected flow devices.
+
+    ``eval(self, t : double, ydot : double[:]) -> None``
+        Responsible for calculating the time derivative of the state ``ydot``
+        (length `n_vars`) at time ``t`` based on the current state of the
+        reactor.
+
+    ``eval_walls(self, t : double) -> None``
+        Responsible for calculating the net rate of volume change `vdot`
+        and the net rate of heat transfer `qdot` caused by walls connected
+        to this reactor.
+
+    ``eval_surfaces(t : double, ydot : double[:]) -> double``
+        Responsible for calculating the time derivative of the surface species
+        ``ydot`` (length: total number of surface species in all surfaces) and
+        the net rate of production of bulk phase species on surfaces. Returns
+        the net mass flux from surfaces into the bulk phase.
+
+    ``component_name(i : int) -> string``
+        Returns the name of the state vector component with index ``i``
+
+    ``component_index(name: string) -> int``
+        Returns the index of the state vector component named ``name``
+
+    ``species_index(name : string) -> int``
+        Returns the index of the species named ``name``, in either the bulk
+        phase or a surface phase, relative to the start of the species terms in
+        the state vector.
+    """
+
+    reactor_type = "DelegatedReactor"
+
+    delegatable_methods = {
+        'initialize': ('initialize', 'void(double)'),
+        'sync_state': ('syncState', 'void()'),
+        'get_state': ('getState', 'void(double*)'),
+        'update_state': ('updateState', 'void(double*)'),
+        'update_surface_state': ('updateSurfaceState', 'void(double*)'),
+        'get_surface_initial_condition': ('getSurfaceInitialCondition', 'void(double*)'),
+        'update_connected': ('updateConnected', 'void(bool)'),
+        'eval': ('eval', 'void(double, double*)'),
+        'eval_walls': ('evalWalls', 'void(double)'),
+        'eval_surfaces': ('evalSurfaces', 'double(double,double*)'),
+        'component_name': ('componentName', 'string(size_t)'),
+        'component_index': ('componentIndex', 'size_t(string)'),
+        'species_index': ('speciesIndex', 'size_t(string)')
+    }
+
+    def __cinit__(self, *args, **kwargs):
+        self.accessor = dynamic_cast[CxxReactorAccessorPtr](self.rbase)
+
+    def __init__(self, *args, **kwargs):
+        assign_delegates(self, dynamic_cast[CxxDelegatorPtr](self.rbase))
```

Comment:
Add comment explaining what is done here (See `delegator.pyx`)

Reply:
Is the suggestion that this should just say "see delegator.pyx"? I think the documentation of the `assign_delegates` method itself is reasonable.

Reply:
I believe a short descriptive comment explaining what is done here would be great. Without providing context, it will take a while for anyone looking over the source code to figure out what is done here: the `delegator` machinery is quite complex so there is a lot that goes on in this line. My "see `delegator.py`" was not necessarily a suggestion for the descriptive comment :laugh: